src/aurelia-view.js

```diff
@@ -1,14 +1,18 @@
 import {Config} from './config';
 import {ViewManager} from './view-manager';
+import {resolvedView} from './decorators/resolved-view';
 
 function configure(aurelia, configCallback) {
   let config = aurelia.container.get(Config);
 
-  configCallback(config);
+  if (configCallback) {
```

Comment:
Another thought, is checking if perhaps the thing provided is an object. That way you could allow a format like this in your project:

```js
export default {
  namespace: {
    map: {
      foo: 'bar'
    }
  }
}
```